webapp/sources/rudder/rudder-web/src/main/webapp/javascript/rudder/rudder-datatable.js

```diff
@@ -1072,6 +1072,18 @@ function createRuleComponentValueTable (contextPath) {
  *   }
  */
 
+function displayProperty(property) {
+ console.log(typeof property)
+ console.log(property)
+ console.log(JSON.stringify(property, undefined, 2))
+ if (typeof property === "object" ) {
+   console.log(property)
```

Comment:
remaining debug code

Reply:
This should be commited in fact ... it was tests for #3404

Reply:
Not be committed ...